pkg/antctl/raw/multicluster/resourceimport.go

```diff
@@ -0,0 +1,179 @@
+	mcsscheme "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/scheme"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+var command *cobra.Command
+
+type resourceImportOptions struct {
+	namespace    string
+	outputFormat string
+	allNamespace bool
+}
+
+var options *resourceImportOptions
+
+var resourceImportExample = strings.Trim(`
+Gel all resource imports of cluster set in default namesapce
+$ antctl get resourceimport
+Get all resource imports of cluster set in all namespace
+$ antctl get resoruceimport -A
+Get all resource imports of specified namespace
+$ antctl get resourceimport -n <NAMESPACE>
+Get all resource imports and print json format
+$ antctl get resourceimport -o json
+`, "\n")
+
+func (o *resourceImportOptions) validateAndComplete() error {
+	if o.allNamespace {
+		o.namespace = metav1.NamespaceAll
+	} else if o.namespace == "" {
+		o.namespace = metav1.NamespaceDefault
+	}
+
+	return nil
+}
+
+func init() {
+	command = &cobra.Command{
+		Use: "resourceimport",
+		Aliases: []string{
+			"resourceimports",
+			"ri",
+		},
+		Short:   "Print Multi-Cluster resource imports",
+		Args:    cobra.MaximumNArgs(1),
+		Example: resourceImportExample,
+		RunE:    runE,
+	}
+	o := &resourceImportOptions{}
+	options = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "namespace of resource import")
+	command.Flags().StringVarP(&o.outputFormat, "output", "o", "", "Output format. Supported formats: json|yaml")
+	command.Flags().BoolVarP(&o.allNamespace, "all-namespace", "A", false, "Get resource of all namespace or not")
+
+	GetCmd.AddCommand(command)
+}
+
+func runE(cmd *cobra.Command, args []string) error {
+	options.validateAndComplete()
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	kubeconfig.GroupVersion = &schema.GroupVersion{Group: "", Version: ""}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	scheme := k8sruntime.NewScheme()
+	err = mcsscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	err = antreamcscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	err = k8sscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: scheme})
+	if err != nil {
+		return err
+	}
+
+	var res []multiclusterv1alpha1.ResourceImport
+	resourceImportList := &multiclusterv1alpha1.ResourceImportList{}
+	err = k8sClient.List(context.TODO(), resourceImportList, &client.ListOptions{Namespace: options.namespace})
+
+	for _, ri := range resourceImportList.Items {
+		if len(args) > 0 && ri.Name == args[0] {
+			res = append(res, ri)
+			break
+		} else if len(args) == 0 {
+			res = append(res, ri)
+		}
+	}
+	if len(res) == 0 {
+		if len(args) == 0 {
+			fmt.Println("No resource found")
```

Comment:
better to return a message like below to allow user to understand it's empty result in a given Namespace.
```
No resources found in kube-system namespace.
```